src/WorkItems/WorkItemModel.cs

```diff
@@ -4,37 +4,41 @@
 using System;
 using System.Collections.Generic;
 
-namespace Microsoft.WorkItemFiling
+namespace Microsoft.WorkItems
 {
-    /// <summary>
-    /// Describes a work item to be filed.
-    /// </summary>
     public class WorkItemModel
     {
-        public string AccountorOrganization { get; set; }
-     
-        public string ProjectOrRepository { get; set; }
+        // TODO: Provide a meaningful representation of work item state in the model.
+        //       https://github.com/microsoft/sarif-sdk/issues/1758
+
+        // TODO: provide a meaningful unique identifier for work items, that allows
+        //       consumers to locate filed items again in the future.
+        // 
+
+        public string OwnerOrAccount { get; set; }
+
+        public string RepositoryOrProject { get; set; }
 
         public string Title { get; set; }
 
-        public string Body { get; set; }
+        public string BodyOrDescription { get; set; }
 
-        public string Description { get; set; }
-        
-        public string Discussion { get; set; }
+        public string CommentOrDiscussion { get; set; }
 
         public string Area { get; set; }
 
         public string Iteration { get; set; }
 
-        public IList<string> Tags { get; set; }
+        public IList<string> Assigness { get; set; }
```

Comment:
>Assigness [](start = 29, length = 9)

`Assignees`! :-)

Reply:
Geez. :)

---
In reply to: [381022088](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381022088) [](ancestors = 381022088)